tests/docker/Dockerfile

```diff
@@ -55,6 +83,20 @@ ENV TZ="/usr/share/zoneinfo/America/Los_Angeles"
 # Do not ask for confirmations when running apt-get, etc.
 ENV DEBIAN_FRONTEND noninteractive
 
+ENV java_base_dir=/opt/jdk
+
+# Copy the JDKs from the build-native-image stage
+COPY --from=build-native-image ${java_base_dir}/11 ${java_base_dir}/11
+COPY --from=build-native-image ${java_base_dir}/17 ${java_base_dir}/17
+
+# This JAVA_HOME path is also used in tests/kafkatest/services/kafka/util.py, 
+# so if you change it here, you should also change it there.
+ENV JAVA_HOME=${java_base_dir}/17
+ENV PATH="$JAVA_HOME/bin:$PATH"
+
+# Create symlinks for keytool
```

Comment:
why we need it?

Reply:
Using the decompression method to install the JDK won’t create the `keytool` command for you. Since testing requires keytool, we'll need to manually bind it.